File: CS/WcfServerModeSource/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Windows.Forms;
using WcfServerModeSourceDemo.ServiceReference1;
using DevExpress.Data.WcfLinq;

namespace WcfServerModeSourceDemo
{
    public partial class Form1 : Form
    {
        AdventureWorksEntities db;
        public Form1()
        {
            InitializeComponent();
            db = new AdventureWorksEntities(new Uri("http://localhost:61617/WcfDataService1.svc"));
        }

        private void Form1_Load(object sender, EventArgs e)
        {
            gridControl1.DataSource = new WcfServerModeSource()
            {
                KeyExpression = "EmployeeID",
                Query = db.Employee
            };
        }
    }
}

File: VB/WcfServerModeSource/Form1.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Linq
Imports System.Text
Imports System.Windows.Forms
Imports WcfServerModeSourceDemo.ServiceReference1
Imports DevExpress.Data.WcfLinq

Namespace WcfServerModeSourceDemo
	Partial Public Class Form1
		Inherits Form
		Private db As AdventureWorksEntities
		Public Sub New()
			InitializeComponent()
			db = New AdventureWorksEntities(New Uri("http://localhost:61617/WcfDataService1.svc"))
		End Sub

		Private Sub Form1_Load(ByVal sender As Object, ByVal e As EventArgs) Handles MyBase.Load
			gridControl1.DataSource = New WcfServerModeSource() With {.KeyExpression = "EmployeeID", .Query = db.Employee}
		End Sub
	End Class
End Namespace
